web-frontend/style/main.css

```diff
@@ -0,0 +1,25 @@
+.CodeMirror {
+    border: 4px solid gray;
+    width: 40%; 
+    margin: auto;
+    font-size: 16px;
+    text-align: left; 
+    text-shadow: none; 
+}
+
+#wrapper {
+    margin-top: 10%;
+}
+
+body {
+    background-color: mediumpurple; 
+    text-align: center;
+    font-family: monospace; 
+    text-shadow: -2px -2px 0 skyblue; 
```

Comment:
This is a really small nit, I'm sorry I'm being picky but we do a more subtle shadow for now? Sorry, it might just be me but it kinda hurt my eyes for the small text